static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,848 @@
+$header-height: 48px;
+$thumb-width: 90px;
+$badge-width: 80px;
+
+.hosted-page {
+    @include f-textSans;
+}
+
+.hosted__glogo {
+    display: inline-block;
+    padding: $gs-baseline/3 $gs-gutter/2;
+    background-color: $guardian-brand;
+    text-align: right;
+}
+
+.hosted__guardian-logo__svg {
+    fill: #ffffff;
+    width: 115px;
```

Comment:
There are still quite a lot of magic numbers in this PR. Do they fit into our grid system at all?